data/events/scripts/monster.lua

```diff
@@ -136,10 +136,8 @@ function Monster:onDropLoot(corpse)
 			if preyLootPercent > 0 then
 				text = text .. " (active prey bonus)"
 			end
-			if (configManager.getBoolean(configKeys.VIP_SYSTEM_ENABLED)
-				and configManager.getBoolean(configKeys.VIP_SYSTEM_LOOT_ENABLED)
-				and player and player:isVip() and vipPercentLoot > 0) then
-				text = text .. " (vip loot bonus " .. (vipPercentLoot * 100) .. "%)"
+			if (vipLootPercent > 0) then
+				text = text .. " (vip loot bonus " .. (vipLootPercent * 100) .. "%)"
```

Comment:
```suggestion
text = text .. " (vip loot bonus " .. (vipLootPercent * 100) .. "%)"
```
Após a mudanção não está ativando a função de vip no player.

Reply:
não é uma correção, não posso fixar um valor de 20% na msg, se na config estiver setada, por exemplo 10%.